src/helpers/__tests__/helpers.test.ts

```diff
@@ -3,132 +3,132 @@
-  const timestamp = moment().subtract(25, 'hours').toISOString();
-  const timeAgo = getTimeDisplayHowLongAgo(timestamp);
-  expect(timeAgo).toBe('a day ago');
+  it('should throw an error when credential has no authentication information', () => {
+    expect(() => {
+      generateAuthType({});
+    }).toThrow('Unknown credential type');
+  });
 });
 
-it('should throw an error if the given timestamp is not a valid date', () => {
-  const timestamp = 'invalid';
-  expect(() => {
-    getTimeDisplayHowLongAgo(timestamp);
-  }).toThrow('Invalid timestamp');
-});
+describe('noopTranslate', () => {
+  it('should format key, value, and components into a string', () => {
+    const key = 'testKey';
+    const value = 'testValue';
+    const components = [
+      React.createElement('div', { key: '1' }, 'Component 1'),
+      React.createElement('span', { key: '2' }, 'Component 2')
+    ];
+
+    const result = helpers.noopTranslate(key, value, components);
+
+    expect(result).toBe(`t(${key}, ${value}, ${components})`);
+  });
+
+  it('should handle array keys and values', () => {
+    const key = ['key1', 'key2'];
+    const value = ['value1', 'value2'];
+
+    const result = helpers.noopTranslate(key, value, []);
 
-it('should return AuthType.UsernameAndPassword when credential has username and password', () => {
-  const mockCredential: CredentialType = {
-    id: 'mockId',
-    name: 'mockName',
-    created_at: new Date(),
-    updated_at: new Date(),
-    cred_type: 'mockCredType',
-    username: 'testUser',
-    password: 'testPassword',
-    ssh_keyfile: '',
-    auth_token: '',
-    ssh_passphrase: '',
-    become_method: '',
-    become_user: '',
-    become_password: '',
-    sources: []
-  };
-  const result = getAuthType(mockCredential);
-  expect(result).toBe(authType.UsernameAndPassword);
+    expect(result).toBe(`t([key1,key2], [value1,value2])`);
+  });
+
+  it('should handle empty components', () => {
+    const key = 'testKey';
+    const value = 'testValue';
+
+    const result = helpers.noopTranslate(key, value, []);
+
+    expect(result).toBe(`t(${key}, ${value})`);
+  });
 });
 
-it('should return AuthType.Token when credential has auth_token', () => {
-  const mockCredential: CredentialType = {
-    id: 'mockId',
-    name: 'mockName',
-    created_at: new Date(),
-    updated_at: new Date(),
-    cred_type: 'mockCredType',
-    username: '',
-    password: '',
-    ssh_keyfile: '',
-    auth_token: 'mockToken',
-    ssh_passphrase: '',
-    become_method: '',
-    become_user: '',
-    become_password: '',
-    sources: []
-  };
-  const result = getAuthType(mockCredential);
-  expect(result).toBe(authType.Token);
+describe('formatDate', () => {
+  it('should format the date correctly in UTC', () => {
+    const date = new Date('2022-01-01T12:00:00Z');
+    const formattedDate = helpers.formatDate(date);
+
+    expect(formattedDate).toEqual('01 January 2022, 12:00 PM UTC');
```

Comment:
required: I missed the string `toEqual` here, let's be consistent and use `toBe` instead